specification/cognitiveservices/ContentSafety/models.tsp

```diff
@@ -40,6 +40,47 @@ enum AnalyzeImageOutputType {
   FourSeverityLevels,
 }
 
+@added(ContentSafety.Versions.v2024_01_30_Preview)
+@doc("The type of batch results storage mode.")
+enum BatchResultsStorageMode {
+  @doc("Merge each result into one file.")
+  CollectiveResultFile,
+
+  @doc("Store each result in a single file.")
+  IndividualResultFiles,
+}
+
+@added(ContentSafety.Versions.v2024_01_30_Preview)
+@doc("The type of batch analysis task status.")
+enum BatchTaskStatus {
+  @doc("The task has not started yet.")
+  NotStarted,
+
+  @doc("The task is in progress.")
+  Running,
+
+  @doc("The task has failed.")
+  Failed,
+
+  @doc("The task has been succeeded.")
+  Succeeded,
+}
+
+@added(ContentSafety.Versions.v2024_01_30_Preview)
+enum ModelStatus {
+  Created,
+  Training,
```

Comment:
Is there any significance between these that a customer can use? Do you start training immediately upon creation, or is it a separate action to start training? If it's automatic, is there sufficient time between these two states that they are worth having?

Reply:
As model training is a long running operation, we will have a separate action to start training. My expectation to the time is in second level, so we add this "Created" status to reduce confusion and make it clearer.